templates/themes/app/pages/search.html.twig

```diff
@@ -8,14 +8,14 @@
             <div class="container">
                 <div class="search-inputs">
                     <div class="input-group">
-                        <input type="text" value="{{ search_text }}" name="searchText" id="Search" class="form-control" placeholder="{{ 'ngsite.search.placeholder'|trans }}" />
+                        <input type="text" aria-label="{{ 'ngsite.search.placeholder'|trans }}" value="{{ search_text }}" name="searchText" id="Search" class="form-control" placeholder="{{ 'ngsite.search.placeholder'|trans }}" />
                         <button type="submit" class="btn btn-sm btn-dark">{{ 'ngsite.search.button'|trans }}<i class="icon-search" aria-hidden="true"></i></button>
```

Comment:
bilo bi dobro i na ovaj button dodati aria-label, nešto tipa "Submit search"

Reply:
done in https://github.com/netgen/media-site/pull/217/commits/63da769837d1bdd0a335ab637d03dd9874751cd5